core/point/point.go

```diff
@@ -21,3 +23,21 @@ func (p *Point) X() int64 { return p.x }
 
 // Y returns the y-value.
 func (p *Point) Y() int64 { return p.y }
+
+// Two Go-game board position/point translation methods
+// 1. '*Point' immutable int point, to SGF(String of two letters) point
+func (pt *Point) ToSGF() string {
+	sgfX := string(rune((pt.X()) + aValue))
+	sgfY := string(rune((pt.Y()) + aValue))
+	// two lower case letters
+	return sgfX + sgfY
+}
+
+// 3. 'p' SGF(two letter string) Point, to '*Point' immutable Int Point
+func NewFromSGF(sgfPt string) *Point {
```

Comment:
What if the input is malformed? I.e., if sgfPt == "abc" or "a" 

Right now what will happen? What should happen?